ray-operator/controllers/ray/utils/constant.go

```diff
@@ -20,6 +20,7 @@ const (
 	RayNodeLabelKey                          = "ray.io/is-ray-node"
 	RayIDLabelKey                            = "ray.io/identifier"
 	RayClusterServingServiceLabelKey         = "ray.io/serve"
+	RayClusterHeadlessServiceLabelKey        = "ray.io/tpu-worker-svc"
```

Comment:
Is this always going to be used by tpu workers? Maybe we will have other uses for the headless service? @kevin85421

Reply:
> Is this always going to be used by tpu workers?

No, I think it is possible to have other uses in the future.